src/main/java/com/apifest/doclet/ParsedEndpoint.java

```diff
@@ -0,0 +1,21 @@
+package com.apifest.doclet;
+
```

Comment:
Please, add ApiFest project license - "Copyright 2013-2015, ApiFest project..." to all new files. Also, add Javadoc to the class.